doc/user-manual/language/polarity.lagda.rst

```diff
@@ -0,0 +1,109 @@
+..
+  ::
+  {-# OPTIONS --polarity #-}
+
+  module language.polarity where
+
+  open import Agda.Builtin.Nat
+
+.. _polarity:
+
+======================
+ Polarity Annotations
+======================
+
+Agda now supports explicitely annotating functions arguments and datatype
+parameters with their polarities, which the positivity checker can then use to
+infer positivity. This experimental feature is hidden behind the
```

Comment:
I don't think "is hidden behind" is the terminology we usually use, I suggest "can be activated with"